skyportal/handlers/api/candidate.py

```diff
@@ -185,6 +186,19 @@ def get(self, obj_id=None):
             description: |
               The sort order for annotations - either "asc" or "desc".
               Defaults to "asc".
+          - in: query
+            name: annotationFilterList
+            nullable: true
+            schema:
+              type: array
+              items:
+                type: string
+            explode: false
+            style: simple
+            description: |
+              Comma-separated string of annotation filter strings formatted as
+              "{origin}:{key}:{value}" for non-numeric fields or
+              "{origin}:{key}:{min}:{max}" for numeric fields
```

Comment:
What about when origin contains a comma ?

Reply:
Good point, I think switching to parsing json like you suggested is the right way to go